assets/app/mixins/window-manager.js

```diff
@@ -72,9 +73,25 @@ export default Ember.Mixin.create({
     this.toggleProperty('windowCollector.onboardApp');
   },
 
+  checkExtension() {
+    if (navigator.userAgent.indexOf('Chrome') !== -1) {
+      window.postMessage({type: 'check'}, '*');
```

Comment:
We should add the event listener prior to sending the message. There should be no race condition here, it's just so the code is more clear.